packages/circle-react-elements/src/components/TransactionStateText/TransactionStateText.stories.ts

```diff
@@ -6,6 +6,22 @@ const meta = {
   title: 'Transaction Components/TransactionStateText',
   component: TransactionStateText,
   tags: ['autodocs'],
+  argTypes: {
+    state: {
+      control: 'select',
+      options: [
```

Comment:
@avkos This approach enhances Storybook controls. Please use the same method for any other components that could benefit from improvement.